commit fe5485b2acf4e2cf5cddc7432b34d96056b1f01b
Author: Volker Krause <vkrause@kde.org>
Date:   Mon Jan 1 18:11:43 2018 +0100

    Don't export kded5 as a CMake target
    
    Summary:
    kded5 is not needed during the build, but exporting it will make the
    corresponding find_package call verify its presence nevertheless. This
    is inconvenient when cross-compiling, where kded5 typically does not
    exist for the host system.
    
    Reviewers: #frameworks
    
    Tags: #frameworks
    
    Differential Revision: https://phabricator.kde.org/D9595

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index 488e386..36d97d5 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -25,7 +25,7 @@ target_compile_definitions(kded5 PRIVATE
 install(TARGETS kdeinit_kded5 EXPORT KDEDTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS})
 
 target_link_libraries( kded5 kdeinit_kded5)
-install(TARGETS kded5 EXPORT KDEDTargets ${KF5_INSTALL_TARGETS_DEFAULT_ARGS} )
+install(TARGETS kded5 ${KF5_INSTALL_TARGETS_DEFAULT_ARGS} )
 
 
 qt5_generate_dbus_interface( kdedadaptor.h org.kde.kded5.xml )
